{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "749fd35b-595f-4281-ba3c-6009f0565505",
   "metadata": {},
   "source": [
    "#  STEP 6: Phylogenetics R&D (Phase 8)\n",
    "\n",
    "**Goal:** To convert our final VCF into an alignment, and then use that alignment to build our final, publication-ready Phylogenetic Tree.\n",
    "\n",
    "**Why (The Final Goal):**\n",
    "This is the scientific climax of the project.\n",
    "1.  **Conversion (The \"Recipe\"):** Our \"Golden File\" (`ANALYSIS_READY.vcf.gz`) is a *list of differences*. Tree-builders like `IQ-TREE` need an *alignment* (a FASTA or PHYLIP file). We will test our new tool (`snp-sites`) to perform this conversion.\n",
    "2.  **Tree Building (The \"R&D\"):** We will test our primary analysis tool (`iqtree`) on this new alignment to generate the final tree."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f1a86653-2d12-4e85-881f-425c9ee766ce",
   "metadata": {},
   "source": [
    "##  1: Handoff from \"The Factory\" (Phase 7 Success)\n",
    "\n",
    "**The Handoff (Done):**\n",
    "We successfully updated our `Snakefile` (V6.0) to automate the `merge` and `filter` \"recipes\" developed in `Notebook 06`. We then executed this final factory step from the terminal:\n",
    "\n",
    "```bash\n",
    "snakemake --cores 1 --rerun-triggers mtime"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e8f2e0b6-0623-4af2-aa92-67aac318ef99",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.19"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
